ansible_runner/defaults.py

```diff
@@ -1,2 +1,6 @@
 default_process_isolation_executable = 'podman'
 default_container_image = 'quay.io/ansible/ansible-runner:devel'
+registry_auth_prefix = 'ansible_runner_registry_'
```

Comment:
Nit: this should be all caps as well since it's a global.
```suggestion
REGISTRY_AUTH_PREFIX = 'ansible_runner_registry_'
```